addons/stock/models/stock_picking.py

```diff
@@ -1459,7 +1459,17 @@ def action_split_transfer(self):
         moves = self.move_ids.filtered(lambda m: m.state not in ('done', 'cancel') and m.quantity != 0)
         backorder_moves = moves._create_backorder()
         backorder_moves += self.move_ids.filtered(lambda m: m.quantity == 0)
-        self._create_backorder(backorder_moves=backorder_moves)
+        backorder = self.with_context(from_manual_backorder=True)._create_backorder(backorder_moves=backorder_moves)
+        backorder.message_post(
+            body=self.env._('Splitted From %s.', self._get_html_link())
```

Comment:
I was suggesting to add a name for the parameter so translator will have better context and I saw "*Splitted*" was red-underlined:
<img width="577" height="72" alt="Screenshot from 2025-12-02 14-44-53" src="https://github.com/user-attachments/assets/69ca425c-50a8-4240-bb4c-47c5da23742e" />
So I checked on Google and it's a irregular verb, the correct past particle is "*split*" (I also made this error a lot of time, even recently :grimacing: )

```suggestion
            body=self.env._('Split from %(original_picking_link)s.', original_picking_link=self._get_html_link())
```